index.js

```diff
@@ -49,8 +49,16 @@ module.exports = function (file, opts) {
     this.push(null)
   }
   function walk (node) {
-    if (node.type === 'TemplateLiteral' &&
-      node.parent.tag && isBelOrYoYo.indexOf(node.parent.tag.name) !== -1) {
+    if (node.type === 'TemplateLiteral' && node.parent.tag &&
+      node.parent.tag.object &&
+      isBelOrYoYo.indexOf(node.parent.tag.object.name) !== -1) {
+      const index = isBelOrYoYo.indexOf(node.parent.tag.object.name)
+      const name = isBelOrYoYo[index]
```

Comment:
could probably find the parent node's name in less characters; might lead to cleaner code